apis/rds/v1alpha1/referencers.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	database "github.com/crossplane/provider-aws/apis/database/v1beta1"
 	network "github.com/crossplane/provider-aws/apis/ec2/v1beta1"
+	"github.com/crossplane/provider-aws/apis/identity/v1beta1"
 	iamv1beta1 "github.com/crossplane/provider-aws/apis/identity/v1beta1"
```

Comment:
```suggestion
	iamv1beta1 "github.com/crossplane/provider-aws/apis/identity/v1beta1"
```

Looks like duplicate import entry.